### ISSUE TITLE
Focus does not go to close cross when details dialog opened

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#552]
Open the Readium Chrome App to the library view.  Open the Add to Library dialog.  Focus should be set into the dialog. The general Readium behavior is to set focus on the close button in the upper right when the dialog is opened. 

#### Expected Behavior
Focus should go to the Close button in the upper right corner of the dialog when the dialog is opened. 

#### Observed behavior
Focus does not go into the dialog until the user presses tab (or clicks into the dialog)

#### Steps to reproduce
1. Open the Readium Chrome App
2. From the library view, select the Add to Library button on the toolbar
3. The Add to Library dialog opens.  
4. Note that focus is not visible within the dialog, press tab to focus the close button in the upper right corner - this button should have been focused immediately when the dialog opened. 

### Product
latest official version available from the Chrome Web Store - 2.23.0
Version of the Chrome browser - Version 51.0.2704.103 (64-bit)
OS and version - OS X 10.11.5

Happens in the cloud reader as well. 

#### Additional information
Will look into fixing this for the managed dialog and About dialog as well.

**Comment by danielweck** [comment]
Fixed by https://github.com/readium/readium-js-viewer/pull/554
